packages/react-lowcode/src/codegen/generation/generators/list/basic-table-generator.ts

```diff
@@ -66,11 +63,13 @@ export class BasicTableGenerator implements TableGenerator
 
                                 if (!this.tableBodyColumnExists(bodyColumns, property)) {
                                     const addHeaderColumn = this.propertyHead(property, this._entity!);
-                                    const addBodyColumn = this.propertyCell(property, this._entity!);
-
+                                    const addBodyColumn = this.propertyCell(property, this._entity!, prefixIterator);                                    
                                     if (columnIndex && columnIndex > 0 && columnIndex < headColumns.length + 1) {
                                         headColumns = [...headColumns.slice(0, columnIndex - 1), ...[addHeaderColumn], ...headColumns.slice(columnIndex - 1)];
                                         bodyColumns = [...bodyColumns.slice(0, columnIndex - 1), ...[addBodyColumn], ...bodyColumns.slice(columnIndex - 1)];
+                                    } else if (columnIndex !== undefined && columnIndex == 0) {
```

Comment:
`columnIndex === 0` ?

Reply:
`columnIndex === 0` means you are inserting on first position in table.